engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/advancedGameSetupScreen/AdvancedGameSetupScreen.java

```diff
@@ -466,6 +468,27 @@ public String get() {
                         });
                     }
                 }
+
+                final UIButton moduleDetails = find("moduleDetails", UIButton.class);
```

Comment:
You should replace `moduleDetails` and other such strings with constants declared at the top of the class.

Reply:
Great idea, I plan to do refactoring of new screens and overall UI next week or two. 
I don't want to do it now, because there are few PRs which will might have conflicts. Better to finish with logic, and only then polish this class.